cmd/app/benchmarks/benchmarks_test.go

```diff
@@ -30,6 +31,7 @@ var (
 				return []byte("ok"), nil
 			},
 		},
+		Log: slog.New(slog.NewTextHandler(ioutil.Discard)),
```

Comment:
Personally I'd prefer the logs were still emitted for the benchmarks

Reply:
Okay, then, I will set them back